lib/reader/src/parser.rs

```diff
@@ -1291,6 +1304,20 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// stack-limit-decl ::= "stack_limit" "=" GlobalVar(gv)
+    fn parse_stack_limit_decl(&mut self) -> Result<GlobalVar> {
+        self.consume();
+        self.match_token(Token::Equal, "expected '=' in stack limit declaration")?;
+        let gv = self.match_gv("expected global variable")?;
+
+        // Collect any trailing comments.
+        self.token();
+        self.claim_gathered_comments(gv);
+        // TODO: Where should I attach claimed comments?
```

Comment:
Where should I attach gathered comments?

Reply:
As things currently stand, comments can only be associated with *entities* (like `gv0` or `ss0` etc.), and the stack_limit isn't an entity. So I think the right thing to do here is to avoid calling `start_gathering_comments` for `stack_limit`, and then you shouldn't have comments to worry about.